src/cljs/org/broadinstitute/firecloud_ui/page/methods_configs_acl.cljs

```diff
@@ -0,0 +1,191 @@
+(ns org.broadinstitute.firecloud-ui.page.methods-configs-acl
+  (:require
+   [dmohs.react :as react]
+   [clojure.string :refer [trim]]
+   [org.broadinstitute.firecloud-ui.common :as common]
+   [org.broadinstitute.firecloud-ui.common.components :as comps]
+   [org.broadinstitute.firecloud-ui.common.style :as style]
+   [org.broadinstitute.firecloud-ui.common.table :as table]
+   [org.broadinstitute.firecloud-ui.endpoints :as endpoints]
+   [org.broadinstitute.firecloud-ui.utils :as utils]
+   ))
+
+
+(defn- get-ordered-name [entity]
+  (clojure.string/join ":"
+    [(entity "namespace")
+     (entity "name")
+     (entity "snapshotId")]))
+
+(defn- is-conf? [e]
+  (contains? e "method"))
+
+(def ^:private access-levels
+  ["READER" "OWNER" "NO ACCESS"])
+
+(defn- build-acl-vec [acl-map]
+  (mapv
+    (fn [k] {:userId k :accessLevel (acl-map k)})
+    (keys acl-map)))
+
+
+(defn- get-agora-method-acl [ns n sid is-conf]
+  {:path (str "/" (if is-conf "configurations" "methods"  ) "/" ns "/" n "/" sid "/permissions"  )
+   :method :get
+   :mock-data
+   (map (fn [i] {(str "user" ) (str "user" i)
+                 (str "roles") (rand-nth
+                                 [["Read"]
+                                  ["Read" "Write" "Create" "Redact" "Manage"]
+                                  []])})
+     (range (inc (rand-int 5))))})
+
+(defn- ui-acl-to-agora-perms [ui-acl-str]
+  (cond
+    (= "NO ACCESS" ui-acl-str) [] ; empty array for no access
+    (= "READER" ui-acl-str) ["Read"] ; just reader
+    (= "OWNER" ui-acl-str) ["Read","Write","Create","Redact","Manage"])) ; all 5 perms
```

Comment:
good catch....I copied the perms from agora....and seemingly either  removed them (or typed them manually) around line 39, but forgot to remove the commas here at line 47